frontend/src/queries/nodes/DataTable/dataTableLogic.ts

```diff
@@ -20,6 +20,7 @@ import equal from 'fast-deep-equal'
 
 export interface DataTableLogicProps {
     key: string
+    nodeKey?: string
```

Comment:
Thinking out loud: reading `nodeKey` here makes it seem like it's referring to an abstract "node", but we're explicitly targeting `dataNode` nodes... So perhaps it's worth standardising all keys everywhere from `key`, `nodeKey`, `tableKey`, etc to something common.

Thinking aobut it, what about `vizKey` and `dataKey` (and/or just `key` to apply to both?), to have some flexibility with regards to the query makeup?

Reply:
I'm not sure I entirely follow but do you mean like this? https://github.com/PostHog/posthog/pull/17187/commits/3f9687033f08e248a2dd22979b2296ff4f14f052

(Please ignore the random last file that slipped in. Removed it in a later commit)

Reply:
Ok, happy with how it is now but can loop back one I get through a few more nodes if something better comes to mind